src/sentry_plugins/phabricator/plugin.py

```diff
@@ -226,8 +228,8 @@ def create_issue(self, request, group, form_data, **kwargs):
         api = self.get_api(group.project)
         try:
             data = api.maniphest.createtask(
-                title=form_data["title"].encode("utf-8"),
-                description=form_data["description"].encode("utf-8"),
+                title=six.text_type(form_data["title"]),
```

Comment:
@scefali @NisanthanNanthakumar Generally you'll want to prefer `django.utils.encoding.force_text`; `str` will result in preservation of the byte literal if the argument is bytes under py3.